src/platforms/hosted/bmda_gpiod.c

```diff
@@ -147,7 +147,7 @@ static bool bmda_gpiod_parse_gpio(const char *name, char *gpio)
 	DEBUG_INFO("gpiochip: %s offset: %u\n", gpio, gpio_off);
 	struct gpiod_line *line = gpiod_line_get(gpio, gpio_off);
 	if (!line) {
-		DEBUG_ERROR("Couldn't get GPIO: %s:%u errno: %d\n", gpio, gpio_off, errno);
+		DEBUG_ERROR("Couldn't get GPIO %s:%u (%d, %s)\n", gpio, gpio_off, errno, strerror(errno));
```

Comment:
we suggest keeping some form of indication of what the number and error string are, this looks fine otherwise.

We suggest something like:
```suggestion
		DEBUG_ERROR("Couldn't get GPIO %s:%u, error %d: %s\n", gpio, gpio_off, errno, strerror(errno));
```